Composite.Workflows/Composite.Workflows.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Composite</RootNamespace>
     <AssemblyName>Composite.Workflows</AssemblyName>
-    <ProjectTypeGuids>{14822709-B5A1-4724-98CA-57A101D1B079};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>

```

Comment:
We still need this "project type guid" here, as it enables workflow editing UI, when the related feature is installed in Visual Studio.
![image](https://user-images.githubusercontent.com/1391705/71079187-0e299b80-218b-11ea-87b4-58fd00b26561.png)

Reply:
Ok, I will rollback this change.
The reason for this change is that after upgrade of vs2019 to the latest version, it is unable to open some projects anymore.
Do you know the reason?

Reply:
Workflow foundation is installed
![image](https://user-images.githubusercontent.com/55481744/71079685-644f0c80-2194-11ea-8f1a-fc038251f3a6.png)

Reply:
Rolled back